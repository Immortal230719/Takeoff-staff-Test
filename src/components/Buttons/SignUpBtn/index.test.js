import React from 'react';
import { shallow } from 'enzyme';
import { SignUpBtn } from './index';

describe('SignUpBtn', () => {
  it('should render correctly', () => {
    const onClick = jest.fn();
    const component = shallow(<SignUpBtn onClick={onClick} className="center">SignUp</SignUpBtn>);

    expect(component.prop('children')).toEqual('SignUp');
    expect(component).toMatchSnapshot();
  });
});
