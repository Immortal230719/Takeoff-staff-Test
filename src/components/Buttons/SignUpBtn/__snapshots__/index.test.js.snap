// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpBtn should render correctly 1`] = `
<WithStyles(ForwardRef(Button))
  className="makeStyles-btnUp-1"
  color="inherit"
  endIcon={
    <UNDEFINED
      color="inherit"
      fontSize="large"
    />
  }
  onClick={[MockFunction]}
  size="large"
  type="submit"
  variant="contained"
>
  SignUp
</WithStyles(ForwardRef(Button))>
`;
